core/storage/collection/gae_models.py

```diff
@@ -76,6 +76,18 @@ def get_deletion_policy():
         """Collection is deleted only if it is not public."""
         return base_models.DELETION_POLICY.KEEP_IF_PUBLIC
 
+    @classmethod
+    def has_reference_to_user_id(cls, user_id):
+        """Check whether CollectionModel snapshots reference user.
```

Comment:
... references the given user.

(Ditto elsewhere.)

Reply:
Done.